code401challenges/src/main/java/stacksandqueues/PseudoQueue.java

```diff
@@ -0,0 +1,38 @@
+package stacksandqueues;
+
+public class PseudoQueue {
+    public Stack stack1;
+    public Stack stack2;
+
+    public PseudoQueue() {
+        this.stack1 = new Stack();
+        this.stack2 = new Stack();
+    }
+
+//    enQueue(x)
```

Comment:
Once you're finished you generally don't want to leave comments / corpse code like this